eng/pipelines/release-go-images-pipeline.yml

```diff
@@ -15,6 +15,15 @@ parameters:
     type: string
     default: nil
 
+  - name: isSecurityRelease
```

Comment:
I think we can make this a boolean now rather than a string?

Reply:
fixed